scheduler/src/cook/mesos/api.clj

```diff
@@ -1094,36 +1098,35 @@
 
 (defn waiting-jobs
   [mesos-pending-jobs-fn is-authorized-fn]
-  (-> (liberator/resource
-        :available-media-types ["application/json"]
-        :allowed-methods [:get]
-        :malformed? (fn [ctx]
-                      (try
-                        (if-let [limit (Integer/parseInt (get-in ctx [:request :query-params "limit"] "1000"))]
-                          (if-not (pos? limit)
-                            [true {::error (str "Limit " limit " most be positive")}]
-                            [false {::limit limit}]))
-                        (catch Exception e
-                          [true {::error (str (.getMessage e))}])))
-        :allowed? (fn [ctx]
-                       (let [user (get-in ctx [:request :authorization/user])]
-                         (if (is-authorized-fn user :read {:owner ::system :item :queue})
-                           true
-                           (do
-                             (log/info user " has failed auth")
-                             [false {::error "Unauthorized"}]))))
-        :handle-forbidden (fn [ctx]
-                            (log/info (get-in ctx [:request :authorization/user]) " is not authorized to access queue")
-                            (render-error ctx))
-        :handle-malformed render-error
-        :handle-ok (fn [ctx]
-                     (-> (map-vals (fn [queue]
-                                     (->> queue
-                                          (take (::limit ctx))
-                                          (map d/touch)))
-                                   (mesos-pending-jobs-fn))
-                         cheshire/generate-string)))
-      ring.middleware.json/wrap-json-params))
+  (liberator/resource
+   :available-media-types ["application/json"]
+   :allowed-methods [:get]
+   :as-response (fn [data ctx] {:body data})
```

Comment:
I'd vote not to, since it's really just a no-op and wouldn't have much other utility.